src/css/custom.scss

```diff
@@ -220,6 +223,12 @@ p img.markdown-image {
   }
 }
 
+// Allow hiding directories in docs-content
+
+.hidden-category {
+  display: none !important;
+}
+
```

Comment:
@yuliiiah do we need this CSS override now that we are keeping tutorials outside of docs?

Reply:
Yup, since the tutorials folder lives under docs-content! If we don't hide it from there, it will be visible in the sidebar 😅